packages/shared/src/components/fields/RadioItem.module.css

```diff
@@ -18,23 +18,23 @@
 
 .item.checked:hover .checkmark, .checked input:focus-visible ~ .checkmark {
   /* stylelint-disable-next-line unit-no-unknown */
```

Comment:
This should no longer be unkown color or is it because of the 3D?

Reply:
Looks like it is because of the 3D as you mentioned.

https://app.circleci.com/pipelines/github/dailydotdev/apps/6094/workflows/69549dd1-13b0-4431-8000-a658efefd36c/jobs/9219